.gitignore

```diff
@@ -1,2 +1,5 @@
 target/
 **/*.rs.bk
+
+# IDEs
+.idea/*
```

Comment:
I think this something that rather belongs in a personal, global `.gitignore` file.